### PR TITLE
Minor Fix of Patient History BreadCrumb

### PR DESCRIPTION
**Pull request by sherrif10**
cc @mozzy11 

https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/48015736/fb1c00c7-e450-4dd7-ab32-cd52cc90591d

### CODE REVIEW

File: frontend/src/languages/en.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "login.msg.password.missing": "A password is required",
     "login.notice.message": "Notice: Access to this service is for authorized personnel only. If you do not have the expressed authorization of the administrator, you must exit now. This organization prohibits unauthorized access, disclosure, duplication, modification, diversion, destruction, loss, misuse, or theft of its information.",
     "error.invalidcredentials": "Username or Password are incorrect",
-    "patient.label.modify" : "Add/Modify Patient" ,
+    "patient.label.modify" :"Modify Patient" ,
```
Comment (sherrif10) [inline frontend/src/languages/en.json:23]: cc @mozzy11

Comment (mozzy11) [reply frontend/src/languages/en.json:23]: @sherrif10 ideally the component can either add a new patient or modify an existing one . 
That why the label was Add/Modify

Comment (sherrif10) [reply frontend/src/languages/en.json:23]: ohh Alright cool.

Comment (mozzy11) [reply frontend/src/languages/en.json:23]: but you need to change it both for the languages

Comment (sherrif10) [reply frontend/src/languages/en.json]: Resolved.

File: frontend/src/languages/en.json [hunk 2]
```diff
@@ -34,7 +34,8 @@
     "login.msg.password.missing": "A password is required",
     "login.notice.message": "Notice: Access to this service is for authorized personnel only. If you do not have the expressed authorization of the administrator, you must exit now. This organization prohibits unauthorized access, disclosure, duplication, modification, diversion, destruction, loss, misuse, or theft of its information.",
     "error.invalidcredentials": "Username or Password are incorrect",
-    "patient.label.modify" : "Add/Modify Patient" ,
+    "patient.label.modify" :"Add Or Modify Patient" ,
+    "patient.label.PatientHistory" :"PatientHistory" ,
```
Comment (mozzy11) [inline frontend/src/languages/en.json:38]: Patient History are two words . 
but the property already exists here https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/languages/en.json#L42

Comment (sherrif10) [reply frontend/src/languages/en.json]: It was an Oversight. Resolved.

**Review by sherrif10 — COMMENTED** [review]
Resolved

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**